opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -319,7 +319,9 @@ enum AggregationTemporality {
 //   (point.flags & FLAG_NO_RECORDED_VALUE) == FLAG_NO_RECORDED_VALUE
 //
 enum DataPointFlags {
-  FLAG_NONE = 0;
```

Comment:
For me this is the worst name, I saw at least 5 times people using equality with this value, so make sure we change this.

Reply:
Does renaming FLAG_NONE -> _FLAG_UNSPECIFIED and addition of comments help?

Reply:
I can get rid of the prefix. Was just trying to make them look special. @open-telemetry/specs-approvers any thoughts?

Reply:
OK, remove the _ prefix and renamed to use _DO_NOT_USE suffix instead.